packages/openapi2-parser/test/schema-test.js

```diff
@@ -754,14 +786,15 @@ describe('JSON Schema to Data Structure', () => {
     it('produces empty array element with empty items', () => {
       const schema = {
         type: 'array',
-        items: {},
+        items: { type: 'string' },
```

Comment:
As lack of type is now handled differently this test would be broken. I've explicitly specified a strng type so that the test assertions are clearer (otherwise the value is enum of 5 types).